PowerUp/app/src/main/res/layout/intro_screen_4.xml

```diff
@@ -0,0 +1,16 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="powerup.systers.com/IntroActivity"
+    android:background="@drawable/screenshot">
+
+    <TextView
+        android:layout_width="@dimen/intro_screen_4_textView_layout_width"
+        android:layout_height="@dimen/intro_screen_4_text_layout_height"
+        android:text="@string/intro_screen_4_text"
+        android:layout_gravity="right|center_vertical"
```

Comment:
Why don't you use `center_horizontal` and `center_vertical` and remove all margins here?